ColorPatterns/Scenes/GameOver/GameOverController.swift

```diff
@@ -29,13 +29,14 @@ final class GameOverController: UIViewController {
     }
 
     @IBAction private func restartGameButtonPressed(_ sender: UIButton) {
-        self.view.window?.rootViewController?.dismiss(animated: true, completion: nil)       
+        //TODO: whoooho, make sure you know what you're doing, plan your navigation stack always and be always sure what is where, you'd need to change it
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/c07cd9a3b7a63e61e86332aeb4fa4cabf8b81c17/ColorPatterns/Scenes/GameOver/GameOverController.swift#L32"></span>TODOs should be resolved (whoooho, make sure you know wh…).<br />
<code>todo</code> <code>GameOverController.swift:32</code></td>
    </tr>
  </tbody>
</table>